.gitignore

```diff
@@ -17,3 +17,4 @@ pdm.toml
 __pypackages__/
 .venv/
 .cache/
+.helix
```

Comment:
Oh, I'll remove this, this should added to your global gitignore. I know I add my own editors to gitignore, but I guess that's the benefit of being a maintainer :stuck_out_tongue: 

```suggestion
```